admin/src/pages/View/components/NavigationManager/List/index.tsx

```diff
@@ -0,0 +1,204 @@
+  const {
+    config: { i18nEnabled },
+  } = useDataManager();
+  const hasAnySelected = !!selected.length;
+  const toggleSelected = useCallback(
+    () =>
+      setState({
+        navigations,
+        selected: hasAnySelected ? [] : navigations.map((n) => n),
+        view: "LIST",
+      }),
+    [setState, navigations, hasAnySelected]
+  );
+  const currentlySelectedSet = useMemo(
+    () => new Set(selected.map(({ id }) => id)),
+    [selected]
+  );
+  const handleSelect = (navigation: Navigation, isSelected: boolean) => () => {
+    setState({
+      navigations,
+      selected: isSelected
+        ? selected.filter(({ id }) => id !== navigation.id)
+        : selected.concat([navigation]),
+      view: "LIST",
+    });
+  };
+  const edit = (navigation: Navigation) => () => {
+    setState({
+      view: "EDIT",
+      navigation,
+      alreadyUsedNames: navigations
+        .map(({ name }) => name)
+        .filter((name) => name !== navigation.name),
+    });
+  };
+  const _delete = (navigations: Array<Navigation>) => () => {
+    setState({
+      view: "DELETE",
+      navigations,
+    });
+  };
+  const deleteSelected = useCallback(_delete(selected), [_delete]);
+
+  return (
+    <>
+      <Grid>
+        <GridItem col={12} paddingBottom={3}>
+          {hasAnySelected ? (
+            <Flex direction="row" gap={1}>
+              <Box paddingRight={3}>
+                {getMessage({
+                  id: "popup.navigation.manage.table.hasSelected",
+                  props: {
+                    count: selected.length,
+                  },
+                })}
+              </Box>
+              <Button onClick={deleteSelected} variant="tertiary">
+                {getMessage("popup.navigation.manage.button.delete")}
+              </Button>
+            </Flex>
+          ) : null}
+        </GridItem>
+      </Grid>
+      <Table>
+        <Thead>
+          <Tr>
+            <Th>
+              <BaseCheckbox
+                onValueChange={toggleSelected}
+                value={hasAnySelected}
+              />
+            </Th>
+            <Th>{getMessage("popup.navigation.manage.table.id")}</Th>
+            <Th>{getMessage("popup.navigation.manage.table.name")}</Th>
+            {i18nEnabled ? (
+              <Th>{getMessage("popup.navigation.manage.table.locale")}</Th>
+            ) : null}
+            <Th>{getMessage("popup.navigation.manage.table.visibility")}</Th>
+            <Th />
+          </Tr>
+        </Thead>
+        <Tbody>
+          {navigations.map((navigation) => (
+            <Tr key={navigation.id}>
+              <Td>
+                <BaseCheckbox
+                  onValueChange={handleSelect(
+                    navigation,
+                    currentlySelectedSet.has(navigation.id)
+                  )}
+                  value={currentlySelectedSet.has(navigation.id)}
+                />
+              </Td>
+              <Td>{navigation.id}</Td>
```

Comment:
The font is not aligned with strapi design. Could be like this ... or with "neutral600"
```suggestion
   <Td>
     <Typography textColor="neutral800">
       {navigation.id}
      </Typography>
    </Td>
```